app/javascript/components/payment-error-button.vue

```diff
@@ -32,24 +39,80 @@
 <script>
 
 import invoiceApi from '../api/invoices';
+import { mapState, mapGetters, mapActions } from 'vuex';
+import utils from '../utils/';
+import cartProducts from './cart-products.vue';
+import _ from 'lodash';
```

Comment:
'_' is defined but never used                                          no-unused-vars<br>Absolute imports should come before relative imports                   import/imports-first